lib/facter/tmp_mounts.rb

```diff
@@ -22,6 +22,8 @@
 simplib__tmp_mount_target_dirs.each do |dir|
   Facter.add("tmp_mount#{dir.gsub('/','_')}") do
     setcode do
+      confine { File.directory?(dir) }
```

Comment:
The original code had `confine :kernel => :linux`.  Is there a reason to switch to this directory confine?

Reply:
The original code only created the fact if the directory existed and then confined based on kernel.  Will just confining based on directory be sufficient?

Reply:
@lnemsick-simp Yep. Windows certainly won't have that directory.